airflow/models/taskinstance.py

```diff
@@ -1632,6 +1634,30 @@ def get(
                 """Get Airflow Variable after deserializing JSON value"""
                 return Variable.get(item, default_var=default_var, deserialize_json=True)
 
+        class ConnectionAccessor:
+            """
+            Wrapper around Connection. This way you can get connections in
+            templates by using ``{{ conn.value.variable_name }}`` or
```

Comment:
It's not a variable name but a conn_id

Reply:
good catch, I'll update this when we are decided on `conn.value.conn_id` vs `conn.conn_id`